### PR TITLE
Bug 1197144 - Make test_import_edit_export_contact.py more robust

### PR DESCRIPTION
**Pull request by mwargers**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e27bf050fa527addee955bf19b05d3eb0f28603c

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/contacts/test_import_edit_export_contact.py [hunk 1]
```diff
@@ -11,8 +11,10 @@ def setUp(self):
         GaiaTestCase.setUp(self)
         curr_time = repr(time.time()).replace('.', '')
 
-        self.contact = MockContact(givenName='Name%s'%curr_time[12:], name='Name%s'%curr_time[12:], familyName='')
+        for contact in self.data_layer.sim_contacts:
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/contacts/test_import_edit_export_contact.py]: I think this deletion needs to be discussed. At the moment we don't delete anything on the SIM card when we run the tests (except a targeted delete in the tearDown of this test). I know some people like @mermi store their contacts in the SIM cards while running the tests.

@mwargers Do you think it's possible to make the test more stable without this deletion?

Comment (npark-mozilla) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/contacts/test_import_edit_export_contact.py]: Same as what jlorenzo says, we do wipe the content of the device at the beginning of each test, but wiping sim sounds a bit risky to me.  AFAIK, we don't wipe the SD card contents either, do we now?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=588930ac77a3d227609b680dde20395787142c5a

**Pull request merged**